### PR TITLE
Create ZoomMapView

### PR DESCRIPTION
**Pull request by mcarcaso**
Demo: https://imgur.com/0QvHQTi
Can be accessed by starting a local server from the foam2 dir:
`python -m SimpleHTTPServer`
and going to:
http://localhost:8000/?model=foam.demos.graphics.ZoomMapViewDemo

### CODE REVIEW

File: src/foam/graphics/GreekView.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
+foam.CLASS({
+  package: 'foam.graphics',
+  name: 'GreekView',
```
Comment (kgrgreer) [inline src/foam/graphics/GreekView.js]: Let's rename this LegendView or MapView or ZoomMapView or PanView or something else. I can't find anywhere online that this is referred to as a GreekView. I have an encyclopedia of visual components at home that I'll check.

File: src/foam/graphics/GreekView.js [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+  extends: 'foam.graphics.CView',
+  requires: [
+    'foam.graphics.CView',
```
Comment (kgrgreer) [inline src/foam/graphics/GreekView.js]: sort requires

File: src/foam/graphics/GreekView.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
+foam.CLASS({
```
Comment (kgrgreer) [inline src/foam/graphics/GreekView.js]: missing copyright

File: src/foam/graphics/GreekView.js [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+      this.add(viewPort);
+      this.add(navContainer);
+      
+    }
+  ]
+});
+
+foam.CLASS({
+  package: 'foam.graphics',
+  name: 'GreekViewDemo',
```
Comment (kgrgreer) [inline src/foam/graphics/GreekView.js]: Move to own file (under different package).

File: src/foam/graphics/GreekView.js (hunk 1 above)
Comment (mcarcaso) [reply src/foam/graphics/GreekView.js]: Went with ZoomMapView

**Pull request merged**